pkg/api/handlers/compat/networks.go

```diff
@@ -312,3 +313,81 @@ func RemoveNetwork(w http.ResponseWriter, r *http.Request) {
 	}
 	utils.WriteResponse(w, http.StatusNoContent, "")
 }
+
+// Connect adds a container to a network
+// TODO: For now this func is a no-op that checks the container name, network name, and
+// responds with a 200.  This allows the call to remain intact.  We need to decide how
+// we make this work with CNI networking and setup/teardown.
+func Connect(w http.ResponseWriter, r *http.Request) {
+	runtime := r.Context().Value("runtime").(*libpod.Runtime)
+
+	var netConnect types.NetworkConnect
+	if err := json.NewDecoder(r.Body).Decode(&netConnect); err != nil {
+		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "Decode()"))
+		return
+	}
+	config, err := runtime.GetConfig()
+	if err != nil {
+		utils.InternalServerError(w, err)
+		return
+	}
+	name := utils.GetName(r)
+	exists, err := network.Exists(config, name)
+	if err != nil {
+		utils.InternalServerError(w, err)
```

Comment:
Is this and no such container actually a 500? Feels like they should be 404

Reply:
Ah, we do that below.

Reply:
ah! it turns out that both container and network are 404s.  Ill fix that.

Reply:
upon review, the code is correct.  Exists returns false, nil if it cannot find the network.  So this error check is actually correct. And the false, nil is handled in the next condition.